src/pages/DetailsPage.js

```diff
@@ -83,14 +83,17 @@ class DetailsPage extends React.PureComponent {
 
     render() {
         let details = lodashGet(this.props.personalDetails, lodashGet(this.props.route.params, 'login'));
+        const login = lodashGet(this.props.route.params, 'login');
```

Comment:
I think it makes sense to keep this in the `if (!details)` check, and inline `avatar: ReportUtils.getAvatar(lodashGet(details, 'avatar', ''), login)` - what do you think?

Reply:
Agreed!